packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.html

```diff
@@ -28,7 +28,7 @@
           ></lightning-button>
         </template>
       </div>
-      <c-quantic-facet-manager engine-id={engineId}>
+      <template lwc:if={disableDynamicNavigation}>
```

Comment:
To avoid duplicating the same logic in the same file and ending up with a big HTML file with some conditions in it, what do you think about having two separate HTML templates to cover these two situations, one template will use the facet manager and the second won't.

Reply:
not a bad idea, I was wondering if that would be an issue in the future though if we start making a template for each config, what if we have config A and config B, then we would need template A & !B, template A & B, template !A & B , template !A & !B ?

Reply:
Not every boolean property will make us create a new template in a component.
 Only the properties that yield big code duplication in the HTML are good candidates for this multiple templates rendering method.

This property seems like a good candidate  since  without multiple templates we would do something like this: 
```
  <template lwc:if={condition}>
     Majority of the HTML of the component  
</template>
  <template ekse={condition}>
     Majority of the HTML of the component  duplicated
</template>
``` 
I think having two separate templates in this situation can make the component a lot easier to read.